Libraries/Drei/input-editor.lisp

```diff
@@ -616,6 +616,9 @@ if stuff is inserted after the insertion pointer."
 
 (defmethod stream-process-gesture ((stream drei-input-editing-mixin)
 				   gesture type)
+  ;; Drei is not interested in the pointer gestures.
+  (when (typep gesture 'pointer-event)
+    (return-from stream-process-gesture nil 'null))
```

Comment:
I thought, generally this was supposed to return `gesture` for further processing. But maybe Drei does things differently.

Reply:
as I've mentioned in the comment, drei assumes that only key press events end up here, that's what caused the issue. this is basically the processing drei "wants", that is to receive only keyboard events.